app/lib/actions/katello/content_view/publish.rb

```diff
@@ -9,7 +9,11 @@ class Publish < Actions::EntryAction
         def plan(content_view, description = "", options = {})
           action_subject(content_view)
           content_view.check_ready_to_publish!
-          version = content_view.create_new_version
+          if options[:minor] && options[:major]
```

Comment:
I think we may need to add some validation at the model level to ensure these are both integer values now that we are allowing customization of them. The major field is an integer in the database but minor is defined as a string which would allow anything. I checked buy didn't see any explicit validations for these fields. Can double check with @beav or @jlsherrill in case I missed them.

Reply:
We probably do need to check that they are numeric and that no existing CVV exists with those versions.  In fact it doesn't even look like there is a unique index on those versions, so that needs to be added to.

Reply:
:+1: to numeric check, I was able to set "five" as the minor string and it resulted in a successful task.

Reply:
@beav Added a check to the model to make sure `:minor` is an integer.